app/Http/Controllers/CarController.php

```diff
@@ -42,19 +46,20 @@ public function store(Request $request)
             'carTable' => 'required|string',
             'description'=>'required|string|max:100',
             'price'=>'required|numeric',
-            'image'=>'required|image|mimes:png,jpg,jpeg,gif|max:2024'
+            'image'=>'required|image|mimes:png,jpg,jpeg,gif|max:2024',
+            'cat_id'=>'required|integer'
```

Comment:
it is better to check that the `category_id` exists in the `categories` table also verify it is `id`
search on how to do so

so how it can make your code better?
what if your database has categories with id from 1 -> 5

| id | category_name |
|----------|----------|
| 1 | ... |
| 2 | ... |
| 3 | ... |
| 4 | ... |
| 5 | ... |

what if i am trying to send `category_id` = `6`, I cause your code to make internal server error
and what if i send `category_id` = `hello`